src/LogicApp/LogicApp/LogicApp.csproj

```diff
@@ -14,4 +14,10 @@
     <PackageReference Include="Microsoft.Azure.Management.Logic" Version="4.1.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Please remove any changes to this file - all of this is taken care of at a higher level.

Reply:
Will do